src/config.ts

```diff
@@ -39,7 +39,7 @@ export default {
   fhirServerConfig: {
     auth: {
       // This servers URI
```

Comment:
I know this isn't within the scope of this PR- just saw this
```suggestion
      // This server's URI
```